brave/brave-core
# Add autofill private tab option

Resolves https://github.com/brave/brave-browser/issues/32096



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

Logins:
- Same as passwords but without saving password.

credit cards:
- Fill and save info on regular profile https://fill.dev/form/credit-card-simple
- Disable the option in the settings
- Open it in Private profile and check.
- Saved info should not be substituted if the option is off.
